internal/app/add/add.go

```diff
@@ -1,90 +1,82 @@
 package add
 
 import (
-	"errors"
 	"fmt"
-	"os"
+	"path/filepath"
 	"slices"
 
-	"github.com/DanWlker/remind/internal/pkg/data"
-	"github.com/DanWlker/remind/internal/pkg/record"
-	"github.com/DanWlker/remind/internal/pkg/shared"
+	"github.com/DanWlker/remind/internal/data"
+	"github.com/DanWlker/remind/internal/record"
+	"github.com/DanWlker/remind/internal/shared"
 )
 
 func AddRun(globalFlag bool, args []string) error {
 	if globalFlag {
-		errAddTodoAndAssociateTo := addTodoAndAssociateTo("", args)
-		if errAddTodoAndAssociateTo != nil {
-			return fmt.Errorf("addTodoAndAssociateTo: %w", errAddTodoAndAssociateTo)
+		if err := addTodoAndAssociateTo("", args); err != nil {
+			return fmt.Errorf("addTodoAndAssociateTo: %w", err)
 		}
 		return nil
 	}
 
-	homeRemCurrProExDir, errHomeRemCurrProExDir := shared.GetHomeRemovedWorkingDir()
-	if errHomeRemCurrProExDir != nil {
-		return fmt.Errorf("helper.GetHomeRemovedCurrentProgramExecutionDirectory: %w", errHomeRemCurrProExDir)
+	dir, err := shared.GetHomeRemovedWorkingDir()
```

Comment:
Prefer short variable names, which paradoxically improve readability most of the time.

As a rule of thumb, the closer together the earliest and the latest use of a variable name are, the shorter it can and should be. (Conversely, when uses of a variable are separated by a lot of space, you need to make the name more descriptive.)